### DISCUSSION TITLE
Next version of florisboard

### DISCUSSION THREAD

**Discussion by MXC48-zz** [florisboard/florisboard#1271] [Q&A]
Hello, I would like to know when version 0.3.14 is planned? 
But is it already programmed?

**Comment by patrickgold** [comment]
See #1235, which is the main discussion area for feedback of the 0.3.14 beta's and new settings screen. As for the development, the beginning was mostly re-writing non-UI stuff which took a few days, currently I am working on transferring and rewriting the settings UI. If you want, you can try out the beta versions of FlorisBoard to follow up on the progress already made.
